sdk-src/src/main/java/com/hpe/adm/nga/sdk/network/google/GoogleHttpClient.java

```diff
@@ -509,6 +518,17 @@ private HttpRequest buildBinaryPostRequest(OctaneHttpRequest.PostBinaryOctaneHtt
         return httpRequest;
     }
 
+    private MultipartContent generateBinaryBulkPostRequest(OctaneHttpRequest.PostBinaryBulkOctaneHttpRequest postBinaryBulkOctaneHttpRequest) {
+        MultipartContent content = new MultipartContent()
+                .setMediaType(new HttpMediaType(HTTP_MEDIA_TYPE_MULTIPART_NAME)
+                        .setParameter(HTTP_MULTIPART_BOUNDARY_NAME, HTTP_MULTIPART_BOUNDARY_VALUE));
+
+        for (Triple<String, InputStream, String> binaryFile : postBinaryBulkOctaneHttpRequest.getBinaryFileInfo()) {
```

Comment:
Maybe List.forEach instead of classic (for : list).

Reply:
Yes